manuscript/chapter1-bio.md

```diff
@@ -129,6 +129,7 @@ I> From [https://en.wikipedia.org/wiki/Web_literacy](https://en.wikipedia.org/wi
 
 This program had a very important mission, fight the digital illiteracy. For a country like Italy with a lot of it, with people who don't know how to use a computer while the smartphones was spreading it was very important.   
 Mozilla for many things works like a startup (I work/ed with them and I am still not liking that way of doing things), starts a lot of prototypes by needs or interests, that can be privacy (like the Encrypt campaign, when FBI wanted Apple to share the encryption keys And Codemoji), Mozilla Pulse, Maker Party, Mozilla WebMaker, Firefox Test Pilot, Firefox Hello and many others that I probably forgot.  
+Also the fact that Mozilla uses a lot of metrics that often are not realistic: like remove features but not understanding that are important for a lot of users. Removing them maybe is a wrong solution, maybe you didn't promoted very well or that features is used by everyone just once every month. The cool part is when a feature is reimplemented after years in a similar way, not just because the code was ugly was needing a refactoring but because is useful and others are doing the same.
```

Comment:
Hm, doesn't Mozilla have an „intent to unship” process? I noticed something on the mailing lists.

Reply:
yes but they are always unship, in my experience this is jut an announcement to let us now about something already decided